cuqi/sampler/_gibbs.py

```diff
@@ -0,0 +1,224 @@
+from cuqi.distribution import JointDistribution
+from cuqi.sampler import Sampler
+from cuqi.samples import Samples
+from typing import Dict, Union
+import numpy as np
+import sys
+
+
+class Gibbs:
```

Comment:
To be subclassed from sampler base class?

Reply:
yes :) #34